docs/installing/advanced_man_install.md

```diff
@@ -54,14 +54,14 @@ Follow these steps:
    ```
    {: codeblock}
 
-2. Query the Docker version to check whether it is recent enough:
+2. Query the Docker or Podman version to check whether it is recent enough:
 
    ```bash
    docker --version
```

Comment:
Will that work for podman, or does it use a different CLI?

Reply:
With the podman-docker RPM, or a simple symlink, all of the `docker` commands are emulated by podman.  I suspect that anyone using podman would know. I considered having both commands but it makes the doc a bit redundant for little gain.